lisa/sut_orchestrator/hyperv/platform_.py

```diff
@@ -334,3 +378,255 @@ def _delete_node(node_ctx: NodeContext, wait_delete: bool) -> None:
                 for node in environment.nodes.list()
             ]
         )
+
+
+class HyperVDevicePoolImpl(BaseDevicePoolImpl):
```

Comment:
`HyperVDevicePool` is enough in the name.

Reply:
Sure, will change the name

Reply:
@squirrelsc Please check the latest push